docs/src/rules/func-call-spacing.md

```diff
@@ -5,7 +5,7 @@ related_rules:
 - no-spaced-func
 ---
 
-This rule was **deprecated** in ESLint v8.53.0. Please use the corresponding rule in [`@stylistic/eslint-plugin-js`](https://eslint.style/packages/js).
+This rule was **deprecated** in ESLint v8.53.0. Please use the [corresponding rule](https://eslint.style/rules/js/func-call-spacing) in [`@stylistic/eslint-plugin-js`](https://eslint.style/packages/js).
```

Comment:
```suggestion
This rule was **deprecated** in ESLint v8.53.0. Please use the [corresponding rule](https://eslint.style/rules/js/function-call-spacing) in [`@stylistic/eslint-plugin-js`](https://eslint.style/packages/js).
```

This rule has been renamed. Both `func-call-spacing` and `function-call-spacing` exist in `@stylistic`, but `func-call-spacing` is marked as deprecated. Shall we link to the new name?

Reply:
I agreed. The goal of my changes is to improve life, why should we do two click for navigation instead of one? x) It's fixed fixed.